app/views/transition_landing_page/components/_youtube_player.html.erb

```diff
@@ -33,4 +33,14 @@
       </a>
     </div>
   <% end %>
+  <details class="govuk-details" data-module="govuk-details">
```

Comment:
Could you use the [details component](https://components.publishing.service.gov.uk/component-guide/details) for this?